src/objects/core/tests/factories.py

```diff
@@ -0,0 +1,43 @@
+import random
+from datetime import date, timedelta
+
+import factory
+from django.contrib.gis.geos import Point
+
+from ..models import Object, ObjectRecord
+from factory.fuzzy import BaseFuzzyAttribute
+
+
+class FuzzyPoint(BaseFuzzyAttribute):
+    def fuzz(self):
+        return Point(random.uniform(-180.0, 180.0),
+                     random.uniform(-90.0, 90.0))
+
+
+class ObjectDataFactory(factory.DictFactory):
+    some_field = factory.Sequence(lambda n: n)
+    name = factory.Faker("name")
+    city = factory.Faker("city")
+    description = factory.Faker("sentence")
+    diameter = factory.LazyAttribute(lambda x: random.randrange(1, 10_000))
+
+
+class ObjectFactory(factory.django.DjangoModelFactory):
+    object_type = factory.Faker("url")
+
+    class Meta:
+        model = Object
+
+
+class ObjectRecordFactory(factory.django.DjangoModelFactory):
+    object = factory.SubFactory(ObjectFactory)
```

Comment:
What is the reason of changing this field? This value is not used in the testcases

Reply:
The purpose is to generate richer data for filtering via data_attrs. More data was needed for the performance tests. Currently the tests will better reside in a separate repository using the locust library (https://github.com/maykinmedia/objects-api-performance).